src/Components/Admins/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Table from '../Shared/Table';
+import styles from './admins.module.css';
+import Button from '../Shared/Button';
+import { getEmployees } from '../../redux/employees/thunks';
+import { getProjects } from '../../redux/projects/thunks';
```

Comment:
```suggestion
import { getEmployees, deleteEmployee } from '../../redux/employees/thunks';
import { getProjects, deleteProject } from '../../redux/projects/thunks';
```
And delete lines 11 and 12

Reply:
thanks